src/browser/wisp_network.js

```diff
@@ -281,28 +273,22 @@ WispNetworkAdapter.prototype.send = function(data)
             const result = await ((await fetch(`https://${this.doh_server}/dns-query`, {method: "POST", headers: [["content-type", "application/dns-message"]], body: packet.udp.data})).arrayBuffer());
             reply.udp.data = new Uint8Array(result);
             this.receive(make_packet(reply));
-
         })();
-
     }
 
     if(packet.ntp) {
+        // TODO: remove when this wisp client supports udp
         handle_fake_ntp(packet, this);
         return;
     }
 
-    // ICMP Ping
-    if(packet.icmp && packet.icmp.type === 8) {
-        handle_fake_ping(packet, this);
-        return;
-    }
-
```

Comment:
I removed the fake ping implementation here, it's not very useful (and possibly confusing) when combined with a real networking proxy.